float32 windspeed_north		# Wind component in north / X direction
float32 windspeed_east		# Wind component in east / Y direction
float32 covariance_north	# Uncertainty - set to zero (no uncertainty) if not estimated
float32 covariance_east		# Uncertainty - set to zero (no uncertainty) if not estimated
float32 tas_innov 		# True airspeed innovation
float32 tas_innov_var 		# True airspeed innovation variance
float32 beta_innov 		# Sideslip measurement innovation
float32 beta_innov_var 		# Sideslip measurement innovation variance
float32 tas_scale 		# Estimated true airspeed scale factor
